plugin/src/test/groovy/com/novoda/test/TestAndroidKotlinProject.groovy

```diff
@@ -0,0 +1,78 @@
+package com.novoda.test
+
+class TestAndroidKotlinProject extends TestProject<TestAndroidKotlinProject> {
+    private static final Closure<String> TEMPLATE = { TestAndroidKotlinProject project ->
+        """
+buildscript {
+    repositories { 
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.0.1'  
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.20'
```

Comment:
This has been added

Reply:
I think you may have left two whitespaces at the end of the line before when adding it:

![image](https://user-images.githubusercontent.com/153802/35433730-603439ce-027c-11e8-9c79-84e473e73fa3.png)

EDIT: actually, there seem to be a few :) Nothing serious. Just my nitpicky eye 💃

Reply:
👁‍🗨